src/BookNavigator/bookmarks/ia-bookmarks.js

```diff
@@ -229,21 +229,23 @@ class IABookmarks extends LitElement {
     return bookmark;
   }
 
-  fetchBookmarks() {
-    return this.api.getAll().then(res => res.text()).then((res) => {
-      let response;
+  async fetchBookmarks() {
+    try {
+      const res = await this.api.getAll();
+      const resText = await res.text();
```

Comment:
These are easier to read if kept together.

```suggestion
      const resText = await this.api.getAll().then(r => r.text());
```